src/dxvk/dxvk_image.h

```diff
@@ -274,6 +274,37 @@ namespace dxvk {
           && this->info().format   == view->info().format;
     }
 
+    /**
+     * \brief Sets render target usage frame number
+     *
+     * The image view will track internally when
+     * it was last used as a render target. This
+     * info is used for async shader compilation.
+     * \param [in] frameId Frame number+     */
+    void setRtBindingFrameId(const uint32_t frameId) {
+      if (frameId != m_rtBindingFrameId) {
+        if (frameId == m_rtBindingFrameId + 1)
+          m_rtBindingFrameCount += 1;
+        else
+          m_rtBindingFrameCount = 0;
+
+        m_rtBindingFrameId = frameId;
+      }
+    }
+
+    /**
+     * \brief Checks for async pipeline compatibility
+     *
+     * Asynchronous pipeline compilation may be enabled if the
+     * render target has been drawn to in the previous frames.
+     * \param [in] frameId Current frame ID
+     * \returns \c true if async compilation is supported
+     */
+    [[nodiscard]] bool getRtBindingAsyncCompilationCompat() const {
+      return m_rtBindingFrameCount >= 5;
```

Comment:
DXVK-GPLALL uses `m_rtBindingFrameCount >= 2` for better Async  performance.
```suggestion
      return m_rtBindingFrameCount >= 2;
```